kyma-project/kyma
# IDP Preset overwritten on Lambda Save

**Description**

Changes proposed in this pull request:

- Fix core configuration (nonexistent property was used)
- Fix lambda edit mode (_issuer_ was not assigned on load)

Testing scenario:
0. On dev, apply changes made in `core/src/app/app.config.ts` (use _gateway_kyma_project_io_version_). It will enable you to display lambda details in APIs view, otherwise request will 404.
1. Create custom IDP preset.
2. Create lambda, choosing HTTP Trigger and your custom identity provider.
3. Go to APIs and select your new lambda. It should display your IDP properly.
4. Get back to lambda details and refresh the page, then just hit 'Save' button.
5. If you check out the lambda on APIs again, then without a fix "issuer" field will be brought back to default.

**Related issue(s)**
[Console](https://github.com/kyma-project/console/pull/1483)
